From c7f816c47fab45129083477454969f4e217be515 Mon Sep 17 00:00:00 2001
From: Giovanni Campagna <gcampagna@src.gnome.org>
Date: Sat, 5 Jan 2013 14:30:01 +0100
Subject: [PATCH] Honor NOCONFIGURE

---
 autogen.sh | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/autogen.sh b/autogen.sh
index 04caa99..3b99247 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -44,5 +44,7 @@ if test -z "$*"; then
 	echo "the $0 command line."
 fi
 
-echo "Running configure..."
-$topsrcdir/configure --enable-maintainer-mode "$@"
+if test -z "$NOCONFIGURE"; then
+    echo "Running configure..."
+    $topsrcdir/configure --enable-maintainer-mode "$@"
+fi
-- 
1.8.0.1

